public/frontend/js/comment.js

```diff
@@ -33,4 +32,34 @@ $(document).on('click','#sendcmt',function(e){
             alert(errors);
         }
     });
-});
\ No newline at end of file
+});
+$(document).on('click','#deletecmt',function(evt){
+    evt.preventDefault(evt);
+    var token = $("[name='_token']").val();
+    var comment_id = $("[name='comment_id']").val();
+    const CREATED = 200;
```

Comment:
[rank B] Why do you need 2 CREATED constant? Please check constant define at line 8